### DISCUSSION TITLE
What does OpenAI's model call for?

### DISCUSSION THREAD

**Discussion by shinichi-a** [langflow-ai/langflow#29] [General]
Are you calling gpt-3.5-turbo? Or is it text-davinci-003?

**Comment by ogabrielluiz** [comment]
text-davinci-003. Issue #17 will add functionality for gpt-3.5-turbo and GPT-4 eventually. 

We also want to add the ability to select the model.

**Reply by shinichi-a** [reply]
Thank you for your reply.
I understand now.
